src/pages/LandingPage/LandingPage.jsx

```diff
@@ -0,0 +1,38 @@
+
+import React from 'react'
+import './landingpage.css'
+import { AsideBar } from '../../components/Asidebar/Asidebar';
+import { Hero } from '../../components/HeroSection/Hero';
```

Comment:
Just a suggestion Instead of importing all components one by one from another component file, make one index.js file and import all the components there and whenever you need these components you can import it from index.js file.